package-lock.json

```diff
@@ -0,0 +1,1311 @@
+{
```

Comment:
This file is added to the root directory of the project where it is not required, remove this